custom/icds_reports/utils/aggregation_helpers/distributed/agg_awc.py

```diff
@@ -283,24 +290,29 @@ def updates(self):
                     ucr.supervisor_id = adolescent_girls_table.supervisor_id AND
                     adolescent_girls_table.month=%(start_date)s
                     )
+                 LEFT JOIN
```

Comment:
Did this not filter for migration at all before?

Reply:
Can you elaborate this point?

Reply:
It looks like this query did not check for migration status before adding this clause. Is that correct?

Reply:
Yeah, it's checking the migration_status before too but it's picking the same from the person cases

Reply:
I see, must have missed that line before.